.github/ISSUE_TEMPLATE/2-enhancement.yml

```
@@ -0,0 +1,30 @@
+name: ✨ Enhancement Request
+description: If you have an idea to improve WooCommerce E2E Tests Boilerplate or need something for development please let us know or submit a Pull Request!
```

We should swap this to be WooCommerce Smooth Generator:

```suggestion
description: If you have an idea to improve WooCommerce Smooth Generator or need something for development please let us know or submit a Pull Request!
```